W-15607886 - RPA Manager release notes 2024-04/3 (#369)
mulesoft/docs-rpa: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc

RPA Manager is the cloud-based control plane for your automation projects.

[[april-24-2024]]
== April 24, 2024

// == What's New
include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+2]

> * Invokable processes can now be published to Salesforce to be called via invocable actions.

Reviewer: ```suggestion
* Invocable run configurations can now be published to Salesforce to be used as invocable actions in Flow.
```

> * Invocable run configurations can now be published to Salesforce to be used as invocable actions in Flow.

Reviewer 2: ```suggestion
* You can now publish invocable run configurations to Salesforce and use them as invocable actions in Flow.
```
